app/src/main/java/de/tum/in/tumcampusapp/component/tumui/roomfinder/NavigationDetailsFragment.kt

```diff
@@ -110,7 +110,7 @@ class NavigationDetailsFragment : BaseFragment<Unit>(
     private fun showNavigationDetailsProperties(navigationDetails: NavigationDetails) {
         navigationDetails.properties.forEach { property ->
             val propertyRow = NavigationPropertyRowBinding.inflate(layoutInflater, binding.propsList, true)
-            propertyRow.propertyName.text = getTranslationForPropertyTitle(property.title)
+            propertyRow.propertyName.text = property.title
             propertyRow.propertyValue.text = property.value
         }
     }
```

Comment:
Perhaps someone with more android-fu can help me here. Is it possible to do this more elegantly with bindings, now that we don't do the translation anymore?

Reply:
I dont know of a better way